### PR TITLE
Update for package installation on guest

### PR DESCRIPTION
**Pull request by hsmj1412**
Probability, the repo local database is broken.
Try to clean it and re-install

Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

File: virttest/libvirt_vm.py [hunk 1]
```diff
@@ -2783,23 +2783,28 @@ def install_package(self, name, ignore_status=False):
...
                 # Just check status is not enough
                 # It's necessary to check if install successfully
                 if status != 0 or session.cmd_status(query_cmd) != 0:
-                    if not ignore_status:
-                        raise virt_vm.VMError("Installation of package %s "
-                                              "failed:\n%s" % (name, output))
-                    else:
-                        logging.error("Installation of package %s failed:"
-                                      "\n%s", name, output)
+                    session.cmd_status_output(clean_cmd, timeout=300)
```
Comment (chunfuwen) [inline virttest/libvirt_vm.py]: please add one comment here why we need do yum clean .

File: virttest/libvirt_vm.py [hunk 2]
```diff
@@ -2783,23 +2783,28 @@ def install_package(self, name, ignore_status=False):
...
+                    status, output = session.cmd_status_output(cmd, timeout=300)
```
Comment (chunfuwen) [inline virttest/libvirt_vm.py]: add one comment here: Retry install required packages

File: virttest/libvirt_vm.py (hunk 1 above)
Comment (hsmj1412) [reply virttest/libvirt_vm.py]: Done

File: virttest/libvirt_vm.py (hunk 2 above)
Comment (hsmj1412) [reply virttest/libvirt_vm.py]: Done

**Comment by hsmj1412** [comment]
@chunfuwen Done

**Pull request merged**